src/troute-nwm/src/nwm_routing/output.py

```diff
@@ -162,6 +192,26 @@ def nwm_output_generator(
     if test:
         flowveldepth.to_pickle(Path(test))
     
+    if wbdyo and not waterbodies_df.empty:
+        
+        LOG.info("- writing t-route flow results to LAKEOUT files")
+        start = time.time()
+        
+        for time_index, i in enumerate(wbdy_df.time.unique()):
```

Comment:
Please write-out LAKEOUT files on coarser time intervals. Here, we are writing them for every 5-minute timestep, however we only need to write files at every `dt * qts_subdivisions` seconds (both of those parameters are entered by the user in the .yaml file under `compute_parameters[forcing_parameters]`). In the case of our testing case, `qts_subdivisions = 12` which indicates that there are 12 timesteps per WRF reporting interval. In other words WRF reports every 60 minutes (300 sec * 12 = 3600 sec = 1 hour), and so this is the frequency we expect to get (and write) data from/to CHRTOUT, and in this case, to LAKEOUT. 

The sheer amount of LAKEOUT files we are writing is one bottleneck.